docs/rules/expiring-todo-comments.md

```diff
@@ -0,0 +1,98 @@
+# Add conditions to TODO comments to make them error
+
+Makes it possible to pass arguments to TODO and FIXME comments to trigger errors.
+
+This rule also implements [`eslint/no-warning-comments`](https://eslint.org/docs/rules/no-warning-comments) behavior so you should disable that in favor of using this.
+
+For projects with legacy branches, such as long term supported older versions, there might be situations where you fix TODOs in the master branch but on the other branches you leave it missing. Choose carefully which projects you would want this feature.
+
+Supported conditions:
```

Comment:
Before listing the conditions, you should better explain what exactly this rule does and what problems it solves.

Reply:
Generally, the docs here could be improved a lot. And include more specifics.

Reply:
Done